CONTRIBUTING.rst

```diff
@@ -34,7 +34,7 @@ We welcome open discussion around improvements---both to the documentation as we
   to make sure it has not already been reported. If the bug has not been reported yet,
   create an :nilearn-gh:`new issue <issues/new/choose>`
   including a `minimal runnable example <https://stackoverflow.com/help/minimal-reproducible-example>`_
-  to showcase it (using Nilearn data) as well as your OS and Nilearn version.
+  to showcase it (using `Nilearn data <https://nilearn.github.io/stable/modules/datasets.html>`_) as well as your OS and Nilearn version.
```

Comment:
I think it is better to use the sphinx reference

https://github.com/nilearn/nilearn/blob/638648ba8b0e8f11cc67991d0d5a36c517cc095b/doc/modules/datasets.rst?plain=1#L1

and let sphinx handle the creation of internal links.

https://www.sphinx-doc.org/en/master/usage/referencing.html#cross-referencing-arbitrary-locations

Reply:
There is also an example on line 39 below